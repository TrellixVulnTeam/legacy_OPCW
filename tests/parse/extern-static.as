
# NOTE: "int" or "uint" should be replaced by something like
#       "std.libc.int" or "std.libc.size_t" so that the compiler will insert
#       the appropriate type for the target platform.

extern static mut errno: int;
extern static stdout: *int8;
