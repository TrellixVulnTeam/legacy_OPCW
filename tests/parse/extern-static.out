ModuleDecl <?> _
  Nodes <?> 
    ExternStaticSlot <?> mut errno
      Ident <?> int
    ExternStaticSlot <?> stdout
      PointerType <?>
        Ident <?> int8
